hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/EnrichedSpanConstants.java

```diff
@@ -15,6 +15,10 @@
   public static final String HEAD_EVENT_ID = "head.event.id";
   public static final String API_EXIT_CALLS_COUNT = "api.exit.calls.count";
   public static final String UNIQUE_API_NODES_COUNT = "unique.apis.count";
+  public static final String GRPC_REQUEST_URL_FORMAT_DOTTED = "grpc.request.url.format.dotted";
```

Comment:
I think we can get rid of the `.format.dotted` part in both the attributes.

Reply:
Done.